AUTHORS

```diff
@@ -198,7 +197,6 @@ Nishant Mittal <mittalnishant14@outlook.com>
 Nisheal John <nisheal.work@gmail.com>
 Nithesh N. Hariharan <nithusha21@yahoo.co.in>
 Nitish Bansal <nitishbansal2297@gmail.com>
-Omshi Samal <omshisamal72@gmail.com>
```

Comment:
Ditto here and elsewhere why are there any deletion in this file? (I was only expecting an addition to this list*)